keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -514,13 +568,13 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: 
-- [ ] API .status
-  - Condition name: 
```

Comment:
Can we also have a condition on the PVC indicating provisioning failure with a human readable message similar to the event listed?

Reply:
Events are best effort with a TTL.  Conditions live with the resource itself.  Provisioning failures seem reasonable to be condition worthy since their lifespan matches with the PVC, right?